static/js/reducers/channel_contributors.js

```diff
@@ -25,7 +25,11 @@ const addContributor = (
   data: Map<string, ChannelContributors>
 ): Map<string, ChannelContributors> => {
   const update = new Map(data)
-  const contributors = update.get(channelName) || []
+  let contributors = update.get(channelName) || []
+  contributors = contributors.filter(
+    _contributor =>
+      _contributor.contributor_name !== contributor.contributor_name
+  )
```

Comment:
Would it not be better to scan the list for a matching contributor (using `R.find` or something like that) and only concat if one isn't found?

Reply:
Sure